src/screens/login/loginScreen.js

```diff
@@ -54,16 +53,15 @@ class LoginScreen extends Component {
               <Text style={allStyles.nextText}>Login</Text>
             </TouchableOpacity>
           </View>
+          
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=6016707)